docs/organization-integration/authentication-tokens.md

```diff
@@ -5,7 +5,7 @@ Using Sigrid CI or the Sigrid API requires an *authentication token*. You can cr
 ## Who should create the token?
 
 - **If you want to use Sigrid CI for an existing system that is already part of your Sigrid account:** Any Sigrid user account with access to that system can create the token. 
-- **If you want to on-board new systems to your Sigrid account:** On-boarding systems requires additional rights. This requires a Sigrid user account that is authorized to access all systems. Sigrid user accounts that can *only* access certain systems *cannot* on-board systems.
+- **If you want to on-board new systems to your Sigrid account:** On-boarding systems requires additional rights. Also see [token permissions for on-boarding new systems](#token-permissions-for-on-boarding-new-systems).
```

Comment:
I guess according to the change in line 49, we can maybe remove the bit: " On-boarding systems requires additional rights"